<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPatientPositioningModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPatientPositioningModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient Positioning</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Parameter">
     <property name="text">
      <string>Parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Patient body:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="qMRMLSegmentSelectorWidget" name="SegmentSelectorWidget_PatientBody" native="true">
          <property name="noneEnabled" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_Beam" native="true">
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLRTBeamNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ParameterSet" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLPatientPositioningNode</string>
           </stringlist>
          </property>
          <property name="noneDisplay" stdset="0">
           <string>Please select a node</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Parameter set:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="ComboBox_TreatmentMachine">
          <item>
           <property name="text">
            <string>Channel25Geometry</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Channel26Geometry_Cabin1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Channel26Geometry_Cabin2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Channel26Geometry_Cabin3</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="CheckBox_FixedReferenceCamera">
          <property name="text">
           <string>Fix a 3D camera within the FixedReference frame</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="PushButton_LoadTreatmentMachine">
          <property name="text">
           <string>Load treatment machine</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Beam:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollisionDetectionCollapsibleButton">
     <property name="text">
      <string>Collision detection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="CollisionDetectionStatusLabel">
        <property name="text">
         <string>No collisions detected.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBox_ForceCollisionDetectionUpdate">
        <property name="text">
         <string>Force update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_PatientTableTopControl">
     <property name="text">
      <string>Patient on TableTop Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_PatientTableTopTranslation">
          <property name="title">
           <string>Patient to TableTop Center Translation (mm)</string>
          </property>
          <property name="collapsed">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_PatientTableTopTranslation">
             <property name="minimum">
              <double>-5000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="CheckBox_RotatePatientHeadFeet">
          <property name="text">
           <string>Rotate Patient Head-Feet</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_PatientTableTopMarkups">
          <property name="title">
           <string>Patient and TableTop markups</string>
          </property>
          <property name="collapsed">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>BaseFixed Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>RAS Offset (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_BaseFixedTranslation">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="coordinates" stdset="0">
           <string>-1500,-1600,1400</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_TableTopControl">
     <property name="text">
      <string>TableTop Control</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Xt offset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_4">
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="suffix">
           <string> deg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_2">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Yt offset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_3">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_5">
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="suffix">
           <string> deg</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_6">
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="suffix">
           <string> deg</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Zt offset</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Xt angle</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Yt angle</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Zt angle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_TableTopRobotControl">
     <property name="text">
      <string>TableTop Robot Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>A1</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>A6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA1">
          <property name="minimum">
           <double>-185.000000000000000</double>
          </property>
          <property name="maximum">
           <double>185.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA2">
          <property name="minimum">
           <double>-140.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="value">
           <double>-90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>A3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>A2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>A5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA3">
          <property name="minimum">
           <double>-120.000000000000000</double>
          </property>
          <property name="maximum">
           <double>155.000000000000000</double>
          </property>
          <property name="value">
           <double>90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>A4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA4">
          <property name="minimum">
           <double>-350.000000000000000</double>
          </property>
          <property name="maximum">
           <double>350.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA5">
          <property name="minimum">
           <double>-125.000000000000000</double>
          </property>
          <property name="maximum">
           <double>125.000000000000000</double>
          </property>
          <property name="value">
           <double>90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="ctkSliderWidget" name="SliderWidget_TableRobotA6">
          <property name="minimum">
           <double>-350.000000000000000</double>
          </property>
          <property name="maximum">
           <double>350.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerPatientPositioningModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ParameterSet</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
